sqe-api-server/Services/TextService.cs

```diff
@@ -197,69 +198,69 @@ private static TextEditionDTO _textEditionToDTO(TextEdition ed, List<EditorInfo>
                 textFragments = ed.fragments.Select(
                         x => new TextFragmentDTO
                         {
-                            textFragmentId = x.textFragmentId,
-                            textFragmentName = x.textFragmentName,
-                            editorId = x.textFragmentAuthor,
+                            textFragmentId = x.TextFragmentId.GetValueOrDefault(),
```

Comment:
For posterity, this was Ingo's code.  He wanted to use nullable values to limit the proliferation of objects (this makes the object quite multipurpose).  I'll keep an eye on that though in case it causes more problems than it solves in the end.